FRENCH/src/translation-terms.md

```diff
@@ -1,68 +1,76 @@
 # Traduction des termes
 
-Voici les principaux termes techniques qui ont été traduits de l'Anglais vers le Français.
+Voici les principaux termes techniques qui ont été traduits de l'anglais vers le
+français.
 
-| Anglais | Français | Observation(s) |
+| Anglais | Français | Observations |
 | ------- | ------ | ------ |
-| ahead-of-time compiled language | langage compilé à la volée | - |
+| ahead-of-time compilation | compilation anticipée | sigle : AOT |

```

Comment:
@joeljpresent : i'm not of all familliar with this term, I don't understood it. Both of translation propositions (mine and your) does not help (me) to understoop what it mean, especially in ch01-02 use. Are you sure about this translation ?

Reply:
**Ahead-of-time**
*[Compilation anticipée](https://fr.wikipedia.org/wiki/Compilation_anticip%C3%A9e)*
The program is fully compiled to machine code into an executable *before* it is run.

**Just-in-time** 
*[Compilation à la volée](https://fr.wikipedia.org/wiki/Compilation_%C3%A0_la_vol%C3%A9e)*
The program is first converted into bytecode, then compiled to machine code *while* it is run.

Your proposition mistranslated *ahead-of-time* as *compilation à la volée* (which actually means *just-in-time*) instead of *compilation anticipée*.

Reply:
Oh yeah, I undestand now, and it's a serious mistranslation ! Thank you !